templates/entry/full.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Full entry template.
+ *
+ * @package AMPConf
+ */
+
+?>
+<article id="post-<?php the_ID(); ?>" <?php post_class( 'entry entry--featured' ); ?>>
+	<figure class="entry__thumbnail">
+		<a href="<?php esc_url( the_permalink() ); ?>">
```

Comment:
Note: `the_permalink()` echoes, so the `esc_url()` here will run on `null`, which won't get printed anyway. I'll change this to just `the_permalink()`, as the return goes through `esc_url()` already.